polyfill/test/instant.mjs

```diff
@@ -331,10 +261,7 @@ describe('Instant', () => {
       equal(nsDiff.microseconds, 0);
       equal(nsDiff.nanoseconds, 86400250250250);
     });
-    it('options may only be an object or undefined', () => {
-      [null, 1, 'hello', true, Symbol('foo'), 1n].forEach((badOptions) =>
-        throws(() => feb21.since(feb20, badOptions), TypeError)
-      );
+    it('options may be an object or undefined', () => {
       [{}, () => {}, undefined].forEach((options) => equal(`${feb21.since(feb20, options)}`, 'PT31622400S'));
```

Comment:
```suggestion
      [() => {}].forEach((options) => equal(`${feb21.since(feb20, options)}`, 'PT31622400S'));
```
`test/built-ins/Temporal/Instant/prototype/since/{smallestunit-undefined.js,options-undefined.js}`